Tools/build/bundle_ensurepip_wheels.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+"""
+Download wheels for :mod:`ensurepip` packages from the Cheeseshop.
```

Comment:
```suggestion
Download wheels for :mod:`ensurepip` packages from PyPI.
```

There is no need to use the legacy name here.

Reply:
https://www.youtube.com/watch?v=Svstp1CWuF0 ;-)